### ISSUE TITLE
Tamed Ferrets & Red Pandas are unbreedable

### ISSUE THREAD

**Issue by ZekeZDev** [EterDelta/CrittersAndCompanions#30]
When a player tames a Ferret or Red Panda the entity is no longer breedable as they are forced to sit instead of passing the interaction. I have managed to fix Ferrets in a fork that I am yet to publish and I will likely make a pull request regarding that same goes for Red Pandas which I am currently working on fixing. 

There is also another issue related to this that I have worked around but not exactly fixed:
```java
this.goalSelector.addGoal(5, new FollowParentGoal(this, 1.0D));
```
This line in the code for both Ferrets and Red Pandas breaks the owner follow code for baby variants. Its hilarious however ultimately a bug. The baby entity will stay in a sitting position however if the parent moves or follows the player due to this Goal the baby will follow the parent scooting around sitting down with no animation playing. I have had a look for examples of having both FollowOwnerGoal and FollowParentGoal however in the base game and other mods FollowParentGoal seems to be omitted from tamable entity's. So that is what I have done in my fork.

**Comment by PhantomFusion1** [comment]
> When a player tames a Ferret or Red Panda the entity is no longer breedable as they are forced to sit instead of passing the interaction. I have managed to fix Ferrets in a fork that I am yet to publish and I will likely make a pull request regarding that same goes for Red Pandas which I am currently working on fixing.
> 
> There is also another issue related to this that I have worked around but not exactly fixed:
> 
> ```java
> this.goalSelector.addGoal(5, new FollowParentGoal(this, 1.0D));
> ```
> 
> This line in the code for both Ferrets and Red Pandas breaks the owner follow code for baby variants. Its hilarious however ultimately a bug. The baby entity will stay in a sitting position however if the parent moves or follows the player due to this Goal the baby will follow the parent scooting around sitting down with no animation playing. I have had a look for examples of having both FollowOwnerGoal and FollowParentGoal however in the base game and other mods FollowParentGoal seems to be omitted from tamable entity's. So that is what I have done in my fork.

Question has there been an offical fix for this?

**Comment by EterDelta** [comment]
Thanks for the bug reports. Everything should be fixed in version **2.0.2**.
The _baby_red_panda.animation_ file was missing a sitting animation, making it look like the panda never sat down.
It's important to note that the `FollowParentGoal` had nothing to do with this issues, as its behavior is superseded by `SitWhenOrderedGoal`, which holds a higher priority and the same Goal flags.
